docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -576,6 +576,10 @@ UDP
   Upstream cluster to which the upstream host belongs to. :ref:`alt_stat_name
   <envoy_v3_api_field_config.cluster.v3.Cluster.alt_stat_name>` will be used if provided.
 
+%RAW_UPSTREAM_CLUSTER%
+  Upstream cluster to which the upstream host belongs to. :ref:`alt_stat_name
+  <envoy_v3_api_field_config.cluster.v3.Cluster.alt_stat_name>` does NOT modify this value.
+
```

Comment:
Just for reference: how about add a parameter to the original %UPSTREM_CLUSTER% rather than adding a new command?

Reply:
Didn't think about that option; might be cleaner. I'll put together an implementation

Reply:
I took a look at it and the separate command is cleaner IMO. All other parameters are format strings or keys/indicies; having a boolean for raw cluster name feels kind of awkward compared to everything else. I don't feel to strongly though if you (or others) think it's better

Reply:
I think either is probably fine, but I'm thinking ultimately for this use case what we probably want is a way to better control what kinds of substitutations etc. are done for logging and stats in a more fine grained way. If we go down this path, use cases where there is a genuine wish to have a different observed name to stats are forced to take (potentially machine generated) cluster names and make them fit a specific shape.

That said, I don't have an in-principle objection to allowing the original cluster name in access logs. In that case, I think `UPSTREAM_CLUSTER_RAW` or `UPSTREAM_CLUSTER_ORIGINAL` or `UPSTEAM_CLUSTER_CANONICAL` would be a better naming scheme.

Reply:
Fine, let's do that based on the suggestion from @htuch :)

Reply:
I like UPSTREAM_CLUSTER_RAW and will go with that 👍🏾 +1 on making stat and access log specification more fine-grained; it was a surprise to see the alt stat name show up in access logs. The extra formatter gets us there in the short term though